<html>
<head>
<title>TestNG:  Cart Page Test</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Cart Page Test</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>4/0/0</td>
</tr><tr>
<td>Started on:</td><td>Sat Jul 26 01:22:43 EDT 2025</td>
</tr>
<tr><td>Total time:</td><td>5 seconds (5658 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.qa.swag.tests.LoginPageTest.userListTest()'><b>userListTest</b><br>Test class: com.qa.swag.tests.LoginPageTest</td>
<td><div><pre>java.lang.AssertionError: expected [4] but found [0]
	at org.testng.Assert.fail(Assert.java:97)
	at org.testng.Assert.assertEqualsImpl(Assert.java:136)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:839)
	at org.testng.Assert.assertEquals(Assert.java:849)
	at com.qa.swag.tests.LoginPageTest.userListTest(LoginPageTest.java:31)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:204)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:443)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
</pre></div><a href='#' onClick='toggleBox("stack-trace983051303", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace983051303'><pre>java.lang.AssertionError: expected [4] but found [0]
	at org.testng.Assert.fail(Assert.java:97)
	at org.testng.Assert.assertEqualsImpl(Assert.java:136)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:839)
	at org.testng.Assert.assertEquals(Assert.java:849)
	at com.qa.swag.tests.LoginPageTest.userListTest(LoginPageTest.java:31)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:204)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:443)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
</pre></div></td>
<td>0</td>
<td>com.qa.swag.tests.LoginPageTest@dd71b20</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.qa.swag.tests.LoginPageTest.LogoExistTest()'><b>LogoExistTest</b><br>Test class: com.qa.swag.tests.LoginPageTest</td>
<td></td>
<td>0</td>
<td>com.qa.swag.tests.LoginPageTest@dd71b20</td></tr>
<tr>
<td title='com.qa.swag.tests.LoginPageTest.doLoginTest()'><b>doLoginTest</b><br>Test class: com.qa.swag.tests.LoginPageTest</td>
<td></td>
<td>0</td>
<td>com.qa.swag.tests.LoginPageTest@dd71b20</td></tr>
<tr>
<td title='com.qa.swag.tests.CartPageTest.getCheckCart()'><b>getCheckCart</b><br>Test class: com.qa.swag.tests.CartPageTest</td>
<td></td>
<td>0</td>
<td>com.qa.swag.tests.CartPageTest@46383a78</td></tr>
<tr>
<td title='com.qa.swag.tests.InventoryPageTest.getCheckifCartUpdated()'><b>getCheckifCartUpdated</b><br>Test class: com.qa.swag.tests.InventoryPageTest</td>
<td></td>
<td>0</td>
<td>com.qa.swag.tests.InventoryPageTest@194d329e</td></tr>
<tr>
<td title='com.qa.swag.tests.InventoryPageTest.getItemCheckinCartTest()'><b>getItemCheckinCartTest</b><br>Test class: com.qa.swag.tests.InventoryPageTest<br>Parameters: Sauce Labs Backpack</td>
<td></td>
<td>0</td>
<td>com.qa.swag.tests.InventoryPageTest@194d329e</td></tr>
<tr>
<td title='com.qa.swag.tests.InventoryPageTest.getItemSortTest()'><b>getItemSortTest</b><br>Test class: com.qa.swag.tests.InventoryPageTest</td>
<td></td>
<td>0</td>
<td>com.qa.swag.tests.InventoryPageTest@194d329e</td></tr>
<tr>
<td title='com.qa.swag.tests.InventoryPageTest.getItemTest()'><b>getItemTest</b><br>Test class: com.qa.swag.tests.InventoryPageTest</td>
<td></td>
<td>0</td>
<td>com.qa.swag.tests.InventoryPageTest@194d329e</td></tr>
<tr>
<td title='com.qa.swag.tests.InventoryPageTest.getLogoTitletest()'><b>getLogoTitletest</b><br>Test class: com.qa.swag.tests.InventoryPageTest</td>
<td></td>
<td>0</td>
<td>com.qa.swag.tests.InventoryPageTest@194d329e</td></tr>
<tr>
<td title='com.qa.swag.tests.CartPageTest.getPageHeaderTitleTest()'><b>getPageHeaderTitleTest</b><br>Test class: com.qa.swag.tests.CartPageTest</td>
<td></td>
<td>0</td>
<td>com.qa.swag.tests.CartPageTest@46383a78</td></tr>
<tr>
<td title='com.qa.swag.tests.InventoryPageTest.getPageTitleTest()'><b>getPageTitleTest</b><br>Test class: com.qa.swag.tests.InventoryPageTest</td>
<td></td>
<td>0</td>
<td>com.qa.swag.tests.InventoryPageTest@194d329e</td></tr>
<tr>
<td title='com.qa.swag.tests.CartPageTest.getPageTitleTest()'><b>getPageTitleTest</b><br>Test class: com.qa.swag.tests.CartPageTest</td>
<td></td>
<td>0</td>
<td>com.qa.swag.tests.CartPageTest@46383a78</td></tr>
<tr>
<td title='com.qa.swag.tests.CartPageTest.getPageUrlTest()'><b>getPageUrlTest</b><br>Test class: com.qa.swag.tests.CartPageTest</td>
<td></td>
<td>0</td>
<td>com.qa.swag.tests.CartPageTest@46383a78</td></tr>
<tr>
<td title='com.qa.swag.tests.InventoryPageTest.getPartialUrlTest()'><b>getPartialUrlTest</b><br>Test class: com.qa.swag.tests.InventoryPageTest</td>
<td></td>
<td>0</td>
<td>com.qa.swag.tests.InventoryPageTest@194d329e</td></tr>
<tr>
<td title='com.qa.swag.tests.InventoryPageTest.getToCartShoppingPage()'><b>getToCartShoppingPage</b><br>Test class: com.qa.swag.tests.InventoryPageTest</td>
<td></td>
<td>0</td>
<td>com.qa.swag.tests.InventoryPageTest@194d329e</td></tr>
<tr>
<td title='com.qa.swag.tests.LoginPageTest.pageTitleTest()'><b>pageTitleTest</b><br>Test class: com.qa.swag.tests.LoginPageTest</td>
<td></td>
<td>0</td>
<td>com.qa.swag.tests.LoginPageTest@dd71b20</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-skipped'>
<tr><td colspan='4' align='center'><b>SKIPPED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.qa.swag.tests.LoginPageTest.userListTest()'><b>userListTest</b><br>Test class: com.qa.swag.tests.LoginPageTest</td>
<td><div><pre>java.lang.AssertionError: expected [4] but found [0]
	at org.testng.Assert.fail(Assert.java:97)
	at org.testng.Assert.assertEqualsImpl(Assert.java:136)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:839)
	at org.testng.Assert.assertEquals(Assert.java:849)
	at com.qa.swag.tests.LoginPageTest.userListTest(LoginPageTest.java:31)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:443)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
</pre></div><a href='#' onClick='toggleBox("stack-trace1195025192", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1195025192'><pre>java.lang.AssertionError: expected [4] but found [0]
	at org.testng.Assert.fail(Assert.java:97)
	at org.testng.Assert.assertEqualsImpl(Assert.java:136)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:839)
	at org.testng.Assert.assertEquals(Assert.java:849)
	at com.qa.swag.tests.LoginPageTest.userListTest(LoginPageTest.java:31)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:443)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
</pre></div></td>
<td>0</td>
<td>com.qa.swag.tests.LoginPageTest@dd71b20</td></tr>
<tr>
<td title='com.qa.swag.tests.LoginPageTest.userListTest()'><b>userListTest</b><br>Test class: com.qa.swag.tests.LoginPageTest</td>
<td><div><pre>java.lang.AssertionError: expected [4] but found [0]
	at org.testng.Assert.fail(Assert.java:97)
	at org.testng.Assert.assertEqualsImpl(Assert.java:136)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:839)
	at org.testng.Assert.assertEquals(Assert.java:849)
	at com.qa.swag.tests.LoginPageTest.userListTest(LoginPageTest.java:31)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:204)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:443)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
</pre></div><a href='#' onClick='toggleBox("stack-trace2066140644", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace2066140644'><pre>java.lang.AssertionError: expected [4] but found [0]
	at org.testng.Assert.fail(Assert.java:97)
	at org.testng.Assert.assertEqualsImpl(Assert.java:136)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:839)
	at org.testng.Assert.assertEquals(Assert.java:849)
	at com.qa.swag.tests.LoginPageTest.userListTest(LoginPageTest.java:31)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:204)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:443)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
</pre></div></td>
<td>0</td>
<td>com.qa.swag.tests.LoginPageTest@dd71b20</td></tr>
<tr>
<td title='com.qa.swag.tests.LoginPageTest.userListTest()'><b>userListTest</b><br>Test class: com.qa.swag.tests.LoginPageTest</td>
<td><div><pre>java.lang.AssertionError: expected [4] but found [0]
	at org.testng.Assert.fail(Assert.java:97)
	at org.testng.Assert.assertEqualsImpl(Assert.java:136)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:839)
	at org.testng.Assert.assertEquals(Assert.java:849)
	at com.qa.swag.tests.LoginPageTest.userListTest(LoginPageTest.java:31)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:204)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:443)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
</pre></div><a href='#' onClick='toggleBox("stack-trace1484580012", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1484580012'><pre>java.lang.AssertionError: expected [4] but found [0]
	at org.testng.Assert.fail(Assert.java:97)
	at org.testng.Assert.assertEqualsImpl(Assert.java:136)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:839)
	at org.testng.Assert.assertEquals(Assert.java:849)
	at com.qa.swag.tests.LoginPageTest.userListTest(LoginPageTest.java:31)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:204)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:443)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
</pre></div></td>
<td>0</td>
<td>com.qa.swag.tests.LoginPageTest@dd71b20</td></tr>
</table><p>
</body>
</html>